fabulaws-config.yml

```diff
@@ -181,6 +202,12 @@
       db-slave: m4.4xlarge
       web: c3.2xlarge # for faster AMI creation (no swap file)
       worker: m4.xlarge
+    prod01:
```

Comment:
e.g.:

      cache: c4.large
      db-master: m4.xlarge
      db-slave: m4.xlarge
      web: c3.large # for faster AMI creation (no swap file)
      worker: m4.large